README.md

```diff
@@ -0,0 +1,189 @@
+
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/rero/rero-ils">
+    <img src=".github/images/logo-global.svg" alt="RERO ILS" width="80" height="80">
+  </a>
+
+<h2 align="center">RERO ILS</h2>
+
+  <p align="center">
+    The elegant solution for heritage, public and school libraries or networks.
+    <br />
+    <a href="https://ils.test.rero.ch/"><strong>Live Demo »</strong></a>
+    ·
+    <a href="https://ils.test.rero.ch/help/home"><strong>User docs »</strong></a>
+    ·
+    <a href="https://github.com/rero/developer-resources"><strong>Developer docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://www.rero.ch/produits/ils">Website</a>
+    ·
+    <a href="https://github.com/rero/rero-ils/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/rero/rero-ils/issues">Request Feature</a>
+  </p>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#rero-ils">RERO ILS</a></li>
+        <li><a href="#built-with">Built with</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#usage">Usage</a>
+      <ul>
+        <li><a href="#demo">Demo</a></li>
+        <li><a href="#features">Features</a></li>
+        <li><a href="#use-rero-ils">Use RERO ILS</a></li>
+      </ul>
+    </li>
+    <li><a href="#getting-started">Getting started</a></li>
+      <ul>
+        <li><a href="#install">Install</a></li>
+        <li><a href="#the-ecosystem">The ecosystem</a></li>
+      </ul>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
+<br />
+
+# About the project
+
+<div align="center">
+  <a href="https://ils.test.rero.ch/">
+    <img src=".github/images/screenshot.png" alt="RERO ILS" style="width:720px">
+  </a>
+</div>
+<br />
+
+## RERO ILS
+
+RERO ILS is a new generation open source [integrated library system](https://en.wikipedia.org/wiki/Integrated_library_system) developed in Switzerland by [RERO+](https://rero.ch/) in collaboration with the Catholic University of Leuven ([UCLouvain](https://uclouvain.be/)). It allows the management of library networks or independent libraries (document acquisition, circulation, cataloguing, search) and offers a public interface for users.
+
+RERO ILS has been under heavy development since 2017 as a replacement for RERO network's legacy software. Its first major release (`v1.0.0`) was published at the end of 2020. The first real-life production instance has been live since June 2021. Since then, it is being actively maintained and developed by a commited team from Switzerland and Belgium, in order to improve upon its current features and satisfy its users' needs.
```

Comment:
```suggestion
RERO ILS has been under heavy development since 2017 as a replacement for RERO network's legacy software. Its first major release (`v1.0.0`) was published at the end of 2020. The first real-life production instance has been live since June 2021. Since then, it is being actively maintained and developed by a committed team from Switzerland and Belgium, in order to improve upon its current features and satisfy its users' needs.
```